src/apps/euler/ethereum/euler.d-token.token-fetcher.ts

```diff
@@ -60,13 +60,6 @@ interface EulerMarketsResponse {
   };
 }
 
-type EulerTokenDataProps = {
-  liquidity: number;
-  interestRate: number;
-  borrowAPY: number;
-  supplyAPY: number;
-};
-
 @Register.TokenPositionFetcher({ appId, groupId, network, options: { includeInTvl: true } })
```

Comment:
That's the option which makes the tvl-fetcher unnecessary : `options: { includeInTvl: true } `